Xamarin.Forms.Platform.Android/Renderers/ScrollViewRenderer.cs

```diff
@@ -424,5 +430,17 @@ void UpdateOrientation()
 				}
 			}
 		}
+
+		void UpdateHorizontalScrollBarVisibility()
+		{
+			if (_hScrollView != null)
+				_hScrollView.HorizontalScrollBarEnabled = _view.HorizontalScrollBarVisibility == ScrollBarVisibility.Always;
+		}
+
+		void UpdateVerticalScrollBarVisibility()
+		{
+			VerticalScrollBarEnabled = _view.VerticalScrollBarVisibility == ScrollBarVisibility.Always ||
```

Comment:
What is `VerticalScrollBarEnabled` for and why not one for horizontal?

Reply:
Thats a good question, that was the suggested property to use over in the Forms issue on this.  I tried to use the Horizontal one, but it didn't do anything.

Seems to me they are nesting two Android scroll views to get vertical and horizontal scrolling.